spec/features/search/search_spec.rb

```diff
@@ -23,7 +23,7 @@
       find(:xpath, "//input[@value='dessert']").click
 
       click_on "Search Recipes"
-      expect(page).to have_current_path("/recipes?diet%5B%5D=vegetarian&ingredients=apples%2C+cinnamon&time=15&type=dessert")
+      expect(page).to have_current_path("/recipes?diet=vegetarian&ingredients=apples%2C+cinnamon&time=15&type=dessert")
```

Comment:
I still don't really understand why our paths need to be the search path, but I guess it is what it is...

Reply:
I think it's just restful convention.. So we start off at the search controller then move to the recipes controller, so we have to pass the params and I think through the url is the easiest way